PDKTZipArchive/PDKTZipArchive.h

```diff
@@ -50,6 +57,42 @@
 
 @end
 
+@interface PDKTZipFileInfo : NSObject
+
+@property (readonly) NSUInteger index;
+
+/**
+ *  The name of the file
+ */
+@property (readonly, strong) NSString *filename;
+
+/**
+ *  The timestamp of the file
+ */
+@property (readonly, strong) NSDate *timestamp;
+
+/**
+ *  The CRC checksum of the file
+ */
+@property (readonly, assign) NSUInteger CRC;
+
+/**
+ *  Size of the uncompressed file
+ */
+@property (readonly, assign) long long uncompressedSize;
+
+/**
+ *  Size of the compressed file
+ */
+@property (readonly, assign) long long compressedSize;
+
+/**
+ *  YES if the file is a directory
```

Comment:
:joy:  I was just following the FileInfo class from another unRAR library. I just copied and pasted the interface and I thought it would be nice to have that documentation... 

But you know I wouldn't done it by myself xDDD